packages/components/src/input-control/reducer/reducer.ts

```diff
@@ -52,6 +52,14 @@ function inputControlStateReducer(
 	composedStateReducers: StateReducer
 ): StateReducer {
 	return ( state, action ) => {
+		// Updates state and returns early when there's no action type. These
+		// are controlled updates and need no exposure to additional reducers.
```

Comment:
When does this happen (action without type)?

Reply:
That would be in what I'm calling the intake effect https://github.com/WordPress/gutenberg/blob/2b14b0fa9cf8a253a884195b9d8bb419542a43f6/packages/components/src/input-control/reducer/reducer.ts#L218-L226

That was dispatching via `reset` in the base PR but in looking for a way to distinguish the update from props I landed on making the bare `dispatch` with no action type. I'd considered implementing another action (or bringing back `update` that used to exist and was used only for this purpose (props coming in)) but I went with the bare `dispatch` because I thought it would make for a little less changes.